packages/fx-core/tests/plugins/resource/aad/unit/generateArmTemplates.test.ts

```diff
@@ -74,11 +74,7 @@ describe("AadGenerateArmTemplates", () => {
         "parameters.json"
       );
       chai.assert.strictEqual(
-        JSON.stringify(
-          expectedResult.Orchestration.ParameterTemplate!.ParameterJson,
-          undefined,
-          2
-        ) + "\n", // workaround test issue caused by prettier
+        JSON.stringify(expectedResult.Orchestration.ParameterTemplate!.ParameterJson, undefined, 2), // workaround test issue caused by prettier
```

Comment:
why we no longer need to append `\n` anymore?

Reply:
since the test file parameters.json files remove the \n at the end of file